Lab10/src/main/java/org/example/Main.java

```diff
@@ -0,0 +1,137 @@
+                        System.out.print("Enter abiturient's minimum average point: ");
+                        double minAveragePoint = scanner.nextDouble();
+                        List<Abiturient> filteredAbiturients = abiturientDAO.filterAbiturientsByHigherAveragePoint(minAveragePoint);
+                        if (!filteredAbiturients.isEmpty())
+                            System.out.println(filteredAbiturients);
+                        else
+                            System.out.println("Abiturients with higher average point than " + minAveragePoint + " not found");
+                    }
+                    case 7 -> {
+                        System.out.println("\t7 - Filter abiturients by hostel needs");
+                        List<Abiturient> filteredAbiturients = abiturientDAO.filterAbiturientsByHostelNeeds();
+                        if (!filteredAbiturients.isEmpty())
+                            System.out.println(filteredAbiturients);
+                        else
+                            System.out.println("Abiturients with hostel needs not found");
+                    }
+                    case 8 -> {
+                        System.out.println("\t8 - Update abiturient");
+                        System.out.print("Enter abiturient's id to update: ");
+                        long idToUpdate = scanner.nextLong();
+                        Abiturient updateToUpdate = Abiturient.initializeAbiturient();
+                        updateToUpdate.setID(idToUpdate);
+                        abiturientDAO.update(updateToUpdate);
+                        System.out.println("Abiturient with id = " + idToUpdate + " successfully updated");
+                    }
+                    case 9 -> {
+                        System.out.println("\t9 - Delete abiturient");
+                        Abiturient abiturientToDelete = Abiturient.initializeAbiturient();
+                        System.out.print("Enter abiturient's id to delete: ");
+                        long idToDelete = scanner.nextLong();
+                        abiturientToDelete.setID(idToDelete);
+                        abiturientDAO.delete(abiturientToDelete);
+                        System.out.println("Abiturient with such parameters and id = " + idToDelete + " successfully deleted");
+                    }
+                    case 10 -> {
+                        System.out.println("\t10 - Delete abiturient by id");
+                        System.out.print("Enter abiturient's id to delete: ");
+                        long idToDelete = scanner.nextLong();
+                        abiturientDAO.deleteById(idToDelete);
+                        System.out.println("Abiturient with id = " + idToDelete + " successfully deleted");
+                    }
+                    case 0 -> { System.out.println("Bye"); }
+                    default -> System.out.println("Wrong option! Try again.");
```

Comment:
Extract case statements into separate methods because of high `switch` complexity